zokrates_js/src/lib.rs

```diff
@@ -20,18 +20,30 @@ use zokrates_core::typed_absy::abi::Abi;
 use zokrates_core::typed_absy::types::ConcreteSignature as Signature;
 use zokrates_field::Bn128Field;
 
+#[wasm_bindgen]
+pub struct CompilationResult {
+    program: Vec<u8>,
+    abi: String,
+}
+
+#[wasm_bindgen]
+impl CompilationResult {
+    pub fn get_program(&self) -> js_sys::Uint8Array {
+        let arr = js_sys::Uint8Array::new_with_length(self.program.len() as u32);
+        arr.copy_from(&self.program);
```

Comment:
Can you explain why copying is required here?

Reply:
There are some concerns around lifetimes if memory is only viewed (see here: https://docs.rs/js-sys/latest/js_sys/struct.Uint8Array.html#unsafety). There are ways to go around this I guess, but we would need to use unsafe code for that.